Formula/influxdb.rb

```diff
@@ -21,25 +21,45 @@ class Influxdb < Formula
 
   depends_on "bazaar" => :build
   depends_on "go" => :build
+  depends_on "make" => :build
```

Comment:
You don't need this if you're using the system `make`, which you are. The formula's `make` executable is called `gmake`, which you don't invoke.